sunpy/net/dataretriever/tests/test_swap.py

```diff
@@ -26,12 +26,12 @@ def test_get_url_for_time_range(timerange, url_start, url_end, level):
     assert urls[-1] == url_end
 
 def test_can_handle_query():
-    assert swap.SWAPClient._can_handle_query(Time('2015/12/30 00:00:00','2015/12/31 00:05:00'), Instrument('swap'), a.Level(1)) == True
-    assert swap.SWAPClient._can_handle_query(Time('2015/12/30 00:00:00','2015/12/31 00:05:00'), Instrument('swap'), a.Level('q')) == True
-    assert swap.SWAPClient._can_handle_query(Time('2015/12/30 00:00:00','2015/12/31 00:05:00'), Instrument('swap'), a.Level('s')) == False
-    assert swap.SWAPClient._can_handle_query(Time('2015/12/30 00:00:00','2015/12/31 00:05:00'), Instrument('swap')) == False
-    assert swap.SWAPClient._can_handle_query(Time('2015/12/30','2015/12/31')) == False
-    assert swap.SWAPClient._can_handle_query(Time('2015/12/30 00:00:00','2015/12/31 00:05:00'), Instrument('eve')) == False
+    assert swap.SWAPClient._can_handle_query(Time('2015/12/30 00:00:00','2015/12/31 00:05:00'), Instrument('swap'), a.Level(1)) is True
+    assert swap.SWAPClient._can_handle_query(Time('2015/12/30 00:00:00','2015/12/31 00:05:00'), Instrument('swap'), a.Level('q')) is True
+    assert swap.SWAPClient._can_handle_query(Time('2015/12/30 00:00:00','2015/12/31 00:05:00'), Instrument('swap'), a.Level('s')) is False
+    assert swap.SWAPClient._can_handle_query(Time('2015/12/30 00:00:00','2015/12/31 00:05:00'), Instrument('swap')) is False
+    assert swap.SWAPClient._can_handle_query(Time('2015/12/30','2015/12/31')) is False
+    assert swap.SWAPClient._can_handle_query(Time('2015/12/30 00:00:00','2015/12/31 00:05:00'), Instrument('eve')) is False
```

Comment:
Remove all the date repetition, set a variable: 
```python
timerange = Time( '2015/12/30 00:00:00', '2015/12/31 00:05:00')
```
and use it everywhere, that makes the code easier to maintain.
Also, 
```python
assert swap... is True
```
is redundant, use:
```python
assert swap...
```
as assert checks whether the next is `True`.